ports/raspberrypi/bindings/rp2pio/StateMachine.h

```diff
@@ -71,4 +71,6 @@ bool common_hal_rp2pio_statemachine_get_rxstall(rp2pio_statemachine_obj_t* self)
 void common_hal_rp2pio_statemachine_clear_rxfifo(rp2pio_statemachine_obj_t *self);
 size_t common_hal_rp2pio_statemachine_get_in_waiting(rp2pio_statemachine_obj_t *self);
 
+void common_hal_rp2pio_statemachine_set_interrupt_handler(rp2pio_statemachine_obj_t *self, void(*handler)(void*), void *arg, int mask);
```

Comment:
For next time, I'd put this in `ports/raspberrypi/common-hal/rp2pio/StateMachine.h` because it's a port-specific API that is unrelated to the Python API. It's a bit weird since this module is itself port-specific.